### DISCUSSION TITLE
Download video never starts, stuck in "queued" state

### DISCUSSION THREAD

**Discussion by wb5rue** [aclap-dev/video-downloadhelper#1008] [Help and Q&A]
### Your question

This just started happening today.  All was working fine.

### Addon & CoApp version and details

Video DownloadHelper
Version 8.2.2.8 (stable)
Browser locale: en-US
Production build
Built on 2024-03-18
Build options: browser=firefox, linuxlic=false, noyt=false, target=mozilla

**Comment by GrampaWildWilly** [comment]
I don't work for VDH.  I'm just another user.

That's a pretty old setup.  What operating system are you running?  What version of the CoApp are you using?  Do you have a VDH license?  Don't post your license.  I'm just asking whether you have one.  I don't suppose I could talk you into getting the latest versions of things.  That would be VDH 9.0.2.14a1 beta & CoApp 2.0.19.  There is 0 chance there will be any bug corrections for your setup but every certainty that active development is in progress on the latest versions.

It might help if you gave some URLs of web pages on which you are finding videos that don't work.  Not names of web sites.  URLs of specific pages where videos reside.  Let's hope they don't require a login, in which case we'll need to try alternative approaches.

Another possibility is that your antivirus or your firewall or both are interfering with things.  You can search this forum for plenty of other discussions of that topic.  What you will find will discuss more recent releases but there were similar problems reported on old releases of VDH as well.
